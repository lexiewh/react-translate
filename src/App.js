function App() {
  return (
    <div className="App">
      <h1>Initial setup</h1>
    </div>
  );
}

export default App;
