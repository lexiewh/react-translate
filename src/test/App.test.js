import App from '../App';
import Enzyme, { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Adapter from 'enzyme-adapter-react-16'

Enzyme.configure({ adapter: new Adapter() })

it('correctly renders enzyme', () => {
  const wrapper = shallow(<App />)

  expect(toJson(wrapper)).toMatchSnapshot()
})