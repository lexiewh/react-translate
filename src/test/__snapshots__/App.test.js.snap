// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctly renders enzyme 1`] = `
<div
  className="App"
>
  <h1>
    Initial setup
  </h1>
</div>
`;
